### PR TITLE
Introduce uninstalling

### PR DESCRIPTION
**Pull request by felixheidecke**
Also includes:
* improved error handling
* consolidated application actions

### CODE REVIEW

**Comment by felixheidecke** [comment]
@DeepDiver1975 This requires the API to return status codes outside the 200 range for it to work!

File: src/App.vue [hunk 1]
```diff
@@ -12,6 +12,10 @@
 	import Navigation from './components/Navigation.vue';
 
 	export default {
+		mounted: function () {
+			this.$store.dispatch('FETCH_APPLICATIONS')
```
Comment (tboerger) [inline src/App.vue]: Uhm... No? It's not used there so it should not be fetched there.

Comment (felixheidecke) [inline src/App.vue]: The _"problem"_ here is, that whenever we change views, the applications will be fetched again and again and again and we don't want that.

App.vue will be mounted only once and whether we start @ in the list, the detail or the update view, so we can minimize API calls.

Re-fetching will occur after application processes such as installing, uninstalling and updating.

Comment (tboerger) [inline src/App.vue]: Than use create, but this approach is totally wrong.

Comment (tboerger) [inline src/App.vue]: > Re-fetching will occur after application processes such as installing, uninstalling and updating.

And this shows a poor API design :)

Comment (felixboehm) [inline src/App.vue]: Please merge for now and change Frontend when API is existing.
No reason for a blocker.

**Pull request merged**